pkg/arvo/gen/vats.hoon

```diff
@@ -1,6 +1,96 @@
 /-  *hood
 :-  %say
+=>
+:: usage: +vats, =options [%option], =short [?]
+|%
++$  options  ?(%suspended %running %blocking %exists %default)
+
++$  short  _|
+
+++  report-running
+  |=  [our=@p now=@da =short]
+  ^-  tang
+  =/  desks  .^((set desk) %cd /(scot %p our)/base/(scot %da now)) ::preps desk variable, set of desks, scries clay for desks 
+  =.  desks
+  %-  silt
+  %+  skim
+  ~(tap in desks) 
+  |=  syd=desk
+  =/  deskstatus  .^(rock:tire:clay %cx /(scot %p our)//(scot %da now)/tire)
+  =(%live -:(~(got by deskstatus) syd))
+  =/  prep  (report-prep our now)
```

Comment:
The code starting here seems to be repeated in all of these new functions.  The "options" field just selects a set of desks, so I think this would be cleaner if reporting on a set of desks was factored separately from using the option to determine that set of desks.  I would also rename "options" to "filter", since it's more specific -- it filters the set of desks.